### ISSUE TITLE
TypeError: "can't access dead object"

### ISSUE THREAD

**Issue by Jack-Works** [calebdwilliams/construct-style-sheets#1]
Envorioment: Firefox **Extension**

Reason:
```js
const iframe = document.createElement('iframe');

iframe.hidden = true;
document.body.appendChild(iframe);
const frameBody = iframe.contentWindow.document.body;
document.body.removeChild(iframe);
```
https://github.com/calebdwilliams/construct-style-sheets/blob/master/adoptedStyleSheets.js#L39
https://github.com/calebdwilliams/construct-style-sheets/blob/master/adoptedStyleSheets.js#L64-L67

The polyfill adds the dom then remove it.

Firefox bans strong reference to DOM object in **WebExtension**, after `.removeChild(iframe)`, `frameBody` become `DeadObject`

```js
frameBody.appendChild(style);
```
https://github.com/calebdwilliams/construct-style-sheets/blob/master/adoptedStyleSheets.js#L88
Then this throws the `TypeError: "can't access dead object"`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Dead_object

**Comment by calebdwilliams** [comment]
Hey @Jack-Works thanks for this. I'll get a new version out ASAP and ping you when it's ready.

**Comment by Jack-Works** [comment]
Cool thanks!

**Comment by calebdwilliams** [comment]
This has been updated. Had some weirdness with my build script, so it published as a version 2.0.0. I'll also port it back to version 1 tonight, but moving to v2 should give you what you need for the time being.
